src/parser/jobs/rpr/modules/ArcaneCircle.tsx

```diff
@@ -47,36 +92,103 @@ export class ArcaneCircle extends RaidBuffWindow {
 			severityTiers: SEVERITIES,
 			adjustCount: this.adjustCount.bind(this),
 		}))
+
+		this.addEvaluator(new DisplayedActionEvaluator([this.data.actions.PERFECTIO]))
 	}
 
-	private adjustCount(window: HistoryEntry<EvaluatedAction[]>, action: TrackedAction): number {
+	private adjustCount(window: HistoryEntry<EvaluatedAction[]>, action: TrackedActionGroup) {
 		const fightTimeRemaining = (this.parser.pull.timestamp + this.parser.pull.duration) - window.start
 
 		if (this.isRushedEndOfPullWindow(window)) {
 			if (fightTimeRemaining < COMMUNIO_BUFFER) {
-				switch (action.action) {
-				case this.data.actions.COMMUNIO:
-					// I hate this linter bypass, but -2 is technically magic and is weird as a const
-					// eslint-disable-next-line @typescript-eslint/no-magic-numbers
-					return -2
+				for (const skill of action.actions) {
+					switch (skill) {
 
-				case this.data.actions.PLENTIFUL_HARVEST:
-					return -1
+					case this.data.actions.CROSS_REAPING:
+					case this.data.actions.VOID_REAPING:
+					case this.data.actions.GRIM_REAPING:
+						return (INVERTOR * REAPINGS_PER_BURST)
 
-				default:
-					return 0
+					case this.data.actions.LEMURES_SLICE:
+					case this.data.actions.LEMURES_SCYTHE:
+						return (INVERTOR * LEMURES_PER_BURST)
+
+					case this.data.actions.SACRIFICIUM:
+						return (INVERTOR * SACRIFICIUM_PER_BURST)
+
+					case this.data.actions.PLENTIFUL_HARVEST:
+						return (INVERTOR * HARVEST_PER_BURST)
+
+					case this.data.actions.COMMUNIO:
+						return (INVERTOR * COMMUNIO_PER_BURST)
+
+					default:
+						return 0
+					}
+				}
+
+				// We have enough time for 1 Communio, but not both
```

Comment:
_Act:_ It looks like you're missing an else branch here.  The for loop underneath this comment is never going to get hit.  

Also, I think if `return -LEMURES_PER_BURST` isn't going to satisfy the linter either, I think I'd rather see suppressing the linter and using -1 * LEMURES_PER_BURST, etc, rather than INVERTOR everywhere.

Reply:
Else added, and -(variable) doesn't appear to be tripping linter, so I'll use that.